x-pack/plugins/security_solution/public/ai_insights/links.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import { AI_INSIGHTS } from '../app/translations';
+import { SecurityPageName, SERVER_APP_ID, AI_INSIGHTS_PATH } from '../../common/constants';
+import type { LinkItem } from '../common/links/types';
+
+export const links: LinkItem = {
+  capabilities: [`${SERVER_APP_ID}.show`],
```

Comment:
> I guess licensing will be implemented when the experimental flag is removed. With the current config, it will always be available with the basic "Security" feature. Is this the desired behavior?

The following message is displayed to users when they don't have an Enterprise license:

![upgrade](https://github.com/elastic/kibana/assets/4459398/39490e46-f6df-4597-9ee1-7765ab4e6cca)

Per our offline discussion, in a future PR we may migrate the branching logic for this check, (currently based on the implementation of a similar check in the assistant), to the new centralized service designed for page-level upsell.
 
> If this new page needs to check the same privileges as the regular AI Assitant, then adding these two lines should make it:
> 
> (assuming `platinum` is the license needed in ESS, I am not sure)
> 
> Or do you plan to use a new sub-feature? If that's the case, I strongly suggest creating it simultaneously with the functionality.

I'll follow up-with @jamesspi re: potentially making `AI Insights` a sub-feature of `Security > Elastic AI Assistant`, using the same technique as https://github.com/elastic/kibana/pull/180732 , which added the sub feature shown in the screenshot below:

![sub-feature](https://github.com/elastic/kibana/assets/4459398/e4ffa737-f625-4ff9-b676-fa6266085d6b)